transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/ExpireDateUtils.java

```diff
@@ -61,7 +65,7 @@ boolean isInvalidDate(@NonNull final String inputDate) {
      * @param inputDate Card Expire Date from view
      * @return date in the server format (yyyy-MM)
      */
-    String convertDateToServerFormat(@NonNull String inputDate) {
+    String convertDateToServerFormat(@NonNull final String inputDate) {
         if (inputDate == null || inputDate.isEmpty()) {
             return "";
         }
```

Comment:
May i know why we are trusting string operations instead of predefined Date parsing classes(like Calender, Date, SimpleDateFormat etc). In my point of view more code, attention needed for date parsing using string operations.